### PR TITLE
[D&P] Unpublish an entry

### PR DESCRIPTION
**Pull request by Convly**
Add the ability to unpublish an entry if it's already published, throws otherwise.

### CODE REVIEW

File: packages/strapi-plugin-content-manager/test/index.test.e2e.js [hunk 1]
```diff
@@ -351,6 +351,29 @@ describe('Content Manager End to End', () => {
       expect(body.message).toBe('Already published');
     });
 
+    test('Unpublish article1, expect article1 to be set to null', async () => {
+      const entry = _.clone(data.articles[0]);
```
Comment (petersg83) [inline packages/strapi-plugin-content-manager/test/index.test.e2e.js:355]: Why do you clone? Can we remove it? Same for  the 2 tests above and the one below

File: packages/strapi-plugin-content-manager/test/index.test.e2e.js [hunk 2]
```diff
@@ -351,6 +351,29 @@ describe('Content Manager End to End', () => {
...
+
+      let { body } = await rq({
+        url: `/content-manager/explorer/application::article.article/unpublish/${entry.id}`,
+        method: 'POST',
+      });
+
```
Comment (petersg83) [inline packages/strapi-plugin-content-manager/test/index.test.e2e.js:361]: Can you add `data.articles[0] = body;` here too please?

**Review by petersg83 — CHANGES_REQUESTED** [review]
Almost goot to me :)

File: packages/strapi-plugin-content-manager/test/index.test.e2e.js (hunk 1 above)
Comment (Convly) [reply packages/strapi-plugin-content-manager/test/index.test.e2e.js:355]: We can :)

File: packages/strapi-plugin-content-manager/test/index.test.e2e.js (hunk 2 above)
Comment (Convly) [reply packages/strapi-plugin-content-manager/test/index.test.e2e.js:361]: Nice catch

**Review by petersg83 — APPROVED** [review]
LGTM :) It can be merged once the tests are done

**Pull request merged**